core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -222,7 +221,7 @@ public void testCreateExistingNamespace() {
 
   @Test
   public void testCreateNamespaceWithProperties() {
-    Assumptions.assumeTrue(supportsNamespaceProperties());
+    assumeThat(supportsNamespaceProperties()).isEqualTo(true);
```

Comment:
let's replace this with `isTrue()` as this reads more fluently

Reply:
Thank you for your suggestion! I will improve the code.